aws/request_pagination_test.go

```diff
@@ -586,6 +586,82 @@ func TestPagination_Standalone(t *testing.T) {
 	}
 }
 
+func TestPagination_Standalone_Pointers(t *testing.T) {
+	type testPageInput struct {
+		NextToken *string
+	}
+	type testPageOutput struct {
+		Value     *string
+		NextToken *string
+	}
+
+	expect := []struct {
+		Value, PrevToken, NextToken *string
+	}{
+		{aws.String("FirstValue"), aws.String("InitalToken"), aws.String("FirstToken")},
+		{aws.String("SecondValue"), aws.String("FirstToken"), aws.String("SecondToken")},
+		{aws.String("ThirdValue"), aws.String("SecondToken"), nil},
```

Comment:
Do we not have a test to test for `""`?

Reply:
Yeah, [TestPagination_Standalone](https://github.com/aws/aws-sdk-go-v2/pull/94/files#diff-c0f28140336767027cefe425e291979cR528) is that testcase. The SDK already had this testcase but the way it [worked](https://github.com/aws/aws-sdk-go-v2/pull/94/files#diff-c0f28140336767027cefe425e291979cR561) prevented the bug from manifesting.